.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "lib/rpc_spec"]
+	path = lib/rpc_spec
+	url = git@github.com:vladmu/rpc_spec.git
```

Comment:
As a reminder,  this was agreed upon and is only temporary. The PR should **not** be merged until this is updated to reflect the correct remote. The rest of this PR can be reviewed as normal.

Reply:
changed to:
```
[submodule "lib/rpc_spec"]
	path = lib/rpc_spec
	url = https://github.com/smartdevicelink/rpc_spec.git
	branch = develop
```